developer_manual/core/unit-testing.rst

```diff
@@ -12,7 +12,13 @@ Getting PHPUnit
 ownCloud >= 10.0
 ^^^^^^^^^^^^^^^^
 
-If you are using ownCloud 10.0 or higher, running ``make test-php`` or ``make`` in your terminal from the root directory will install a local version of PHPUnit inside of ``lib/composer`` and run the test suite.
+If you are using ownCloud 10.0 or higher, running ``make test-php`` or ``make`` in your terminal from the root directory will install a local version of PHPUnit at ``<webroot>/lib/composer/phpunit/phpunit``.
+
+Note
+
+- To run test suites with the provided PHPUnit version, change to the directory where you have located your tests and call ``<webroot>/lib/composer/phpunit/phpunit/phpunit`` plus optinal parameters.
```

Comment:
This is only valid for apps. In core you just run "make test-php" to run the tests.